packages/openapi-ts/src/types/config.d.ts

```diff
@@ -28,15 +28,7 @@ export interface UserConfig {
    *
    * Alternatively, you can define a configuration object with more options.
    */
-  input:
-    | `https://get.heyapi.dev/${string}/${string}`
-    | `${string}/${string}`
-    | `readme:@${string}/${string}#${string}`
-    | `readme:${string}`
-    | `scalar:@${string}/${string}`
-    | (string & {})
-    | (Record<string, unknown> & { path?: never })
-    | Input;
+  input: InputPath | Input | (InputPath | Input)[];
```

Comment:
use readonly arrays please

```suggestion
  input: InputPath | Input | ReadonlyArray<InputPath | Input>;
```